### PR TITLE
[skip ci] WIP populate scratch with minimum OS structure

### LINKED ISSUE #489: Populate scratch with the minimum OS structure
There are some core files and directories that need to exist for the containers to function correctly.
For Linux these are:
/etc/hostname
/etc/hosts
/etc/resolv.conf
/proc
/sys
/lib/modules - needed for bindmount of appropriate kernel modules for insmod
/usr/lib/iptables - possibly needed, but needs confirmation

/.tether

These should be created on scratch so that their presence doesn't show up in image diffs.
The files are present so that they can be used as bindmount targets, not to be populated with specific data.

### PR DESCRIPTION
**Pull request by chengwang86**
fixes #489

### CODE REVIEW

File: isos/bootstrap/bootstrap [hunk 1]
```diff
@@ -76,11 +76,30 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     mkdir -p ${MOUNTPOINT}/usr/lib
     cp -r /usr/lib/iptables ${MOUNTPOINT}/usr/lib/
 
+    mkdir -m 755 -p ${MOUNTPOINT}/.tether/etc ${MOUNTPOINT}/.tether/lib/modules ${MOUNTPOINT}/.tether/usr/lib/iptables
+    mkdir -m 555 -p ${MOUNTPOINT}/.tether/proc ${MOUNTPOINT}/.tether/sys
+
+    touch ${MOUNTPOINT}/.tether/etc/hostname ${MOUNTPOINT}/.tether/etc/hosts ${MOUNTPOINT}/.tether/etc/resolv.conf
```
Comment (jzt) [inline isos/bootstrap/bootstrap]: Curious:  Is it necessary to touch these files in `${MOUNTPOINT}/.tether`?  I know they need to be created on scratch, but doesn't the tether create them here automatically?

File: isos/bootstrap/bootstrap [hunk 2]
```diff
@@ -76,11 +76,30 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
...
     systemctl switch-root ${MOUNTPOINT} /.tether/tether 2>&1
     echo "switched to the new mount"
+
+    sleep 10s
+
+    mount --bind /.tether/etc/hostname /etc/hostname
+    mount --bind /.tether/etc/hosts /etc/hosts
+    mount --bind /.tether/etc/resolv.conf /etc/resolv.conf
+
+    sleep 1m
```
Comment (jzt) [inline isos/bootstrap/bootstrap]: Sorry for commenting on your WIP :) Just curious if this 1m sleep is just for testing.

Comment (chengwang86) [inline isos/bootstrap/bootstrap]: @jzt Yes both `sleep` are just used for my own testing. Caglar told me that the only way to check/debug the `bootstrap` was to watch the screen output of the containerVM from vsphere web client. Unless I add enough `sleep` time between cmds, the output from the photonOS would flush all the useful information I want to see. 

However, in my local test, it seemed like the extra sleep time didn't work.

File: isos/bootstrap/bootstrap (hunk 1 above)
Comment (chengwang86) [inline isos/bootstrap/bootstrap]: @jzt tether only creates these files under `/etc` but not `.tether/etc` which is why I need to create them manually in bootstrap.

**Comment by jzt** [comment]
Should this PR be closed in favor of #5493?

**Comment by chengwang86** [comment]
I will keep this PR for now for my reference and close it shortly.

**Comment by chengwang86** [comment]
This is not needed anymore.

**Pull request closed without merging**